docs/kpi-options/deployment-tutorial.md

```diff
@@ -0,0 +1,130 @@
+---
+title: KPI Options Deployment Tutorial
+sidebar_label: Deployment Tutorial
+---
+
+The process to launch and manage KPI Options on UMA is surprisingly simple. This tutorial will demonstrate how to deploy a KPI Option using the Long Short Pair (LSP) contract and the UMA `launch-lsp` repository.
+
+## Deployment process
+
+The following is an overview of the deployment steps below.
+
+1. LSP Prerequisites
```

Comment:
This is a title, while the others are actions. Maybe just `Setup LSP Deployment Prerequisites`

Reply:
Updated for range tokens, success tokens, and KPI options